package.json

```diff
@@ -97,10 +98,12 @@
     "mdn-polyfills": "^5.16.0",
     "memoize-one": "^5.0.2",
     "moment": "^2.24.0",
+    "node-vibrant": "^3.1.5",
     "preact": "^8.4.2",
     "preact-compat": "^3.18.4",
     "react-big-calendar": "^0.20.4",
     "regenerator-runtime": "^0.13.2",
+    "resize-observer": "^1.0.0",
```

Comment:
Oh Safari :( https://caniuse.com/#search=resize

Reply:
I can probably use this if needed. 
     ` window.addEventListener("resize", this._debouncedResizeListener);`

Reply:
https://www.npmjs.com/package/resize-observer#as-a-polyfill

Reply:
@bramkragten I used the method you described in the Drag and Drop PR. Which is the Polyfill method. Should this be good to go?

Reply:
yes